data/libs/systems/hireling.lua

```diff
@@ -313,6 +313,7 @@ function Hireling:save()
 	sql = sql .. " WHERE `id`=" .. tostring(self.id)
 
 	db.query(sql)
+	return true
```

Comment:
this function was void. if the DB does not execute the query, will it return false?

Reply:
I put it as a precaution, if the bank happens to drop the connection, trying to execute the query will fail, so by default it returns false so as not to have a problem returning nil again.